stdlib/config-api/src/main/ballerina/config/config.bal

```
@@ -97,17 +85,11 @@ public function getAsFloat(@sensitive string key, float default = 0.0) returns f
     if (contains(key)) {
         var value = get(key, FLOAT);
 
-        match value {
-            float floatVal => return floatVal;
-            int|string|boolean|map|any[]|() => {
-                error err = error("Invalid value. Expected a 'float'.");
-                panic err;
-            }
-            error err => {
-                //map data = { cause: err };
-                error e = error("Invalid value. Expected a 'float'.");
-                panic e;
-            }
+        if (value is float) {
+            return value;
+        } else {
+            error err = error("Invalid value. Expected a 'float'.");
+            panic err;
```

Shouldn't we return an error in this case ?